Сетка страниц и крупных блоков на флексах

ParovatkinA opened the pull request:
---
:mortar_board: [Сетка страниц и крупных блоков на флексах](https://up.htmlacademy.ru/htmlcss/33/user/302695/tasks/7)

:boom: https://htmlacademy-htmlcss.github.io/302695-sedona-33/6/

ParovatkinA committed:
задает тегу html значение height 100%

ParovatkinA committed:
добавляет свойство тегу body

ParovatkinA committed:
поворачивает ось и располагает блоки сверху вниз

ParovatkinA committed:
устанавливает минимальную высоту элемента body

ParovatkinA committed:
добавляет класс к блоку <main>

ParovatkinA committed:
делает жадным блок .main-container

ParovatkinA committed:
добавляет размеры и расставляет блок навигации по центру

ParovatkinA committed:
Сбрасывает отступы у списков ссылок в навигации

ParovatkinA committed:
задает размеры меню навигации по макету

ParovatkinA committed:
Задает пользовательскому меню максимальную ширину и сдвигает к правому краю

ParovatkinA committed:
Оборачивает контент сайта в контейнер

для того, чтобы поставить контент по центру.

ParovatkinA committed:
Ставит картинку логотипа по центру

ParovatkinA committed:
Удаляет отступы у списка преимуществ

ParovatkinA committed:
Оборачивает футер в контейнер

для центрирования

ParovatkinA committed:
Разрешает перенос флекс элементов на новый ряд

flex-wrap: wrap

ParovatkinA committed:
прописывает ширину флекс элементов

ParovatkinA committed:
задает картинкам из причин 1 и 5 местоположение

ParovatkinA committed:
Задает ширину флекс элемента социальные сети в футере

ParovatkinA committed:
Задает ширину флекс-элемента subtitle в футере и отступ справа

ParovatkinA committed:
Задает ширину и отступ справа логотипу в футере

ParovatkinA committed:
Корректирует ширину флекс элемента в футере

ParovatkinA committed:
Удаляет ошибочно введенный знак

ParovatkinA committed:
создает флекс блок для секций поиск отеля и рассылки

ParovatkinA committed:
добавляет разметку и стили с главной страницы

ParovatkinA committed:
временно скрывает заголовок

ParovatkinA committed:
задает флексовую модель для блока catalog-products

ParovatkinA committed:
Задает размер блоку фильтра и определяет направление флекс бокса сверху вниз

ParovatkinA committed:
добавляет обертку для основной части страницы каталога

ParovatkinA committed:
добавляет флексовую модель для блока рассылки на странице каталога

mpodaniev commented:
1. Одной строчкой стилей этот элемент уже можно было разместить по сетке:
![image](https://user-images.githubusercontent.com/8547830/153491524-6403512f-8f3f-42de-8c2e-ed6a68321054.png)

2.Футер должен находится по центру: 
![image](https://user-images.githubusercontent.com/8547830/153491631-2ab3179d-0882-4ed1-a6ff-f861305276a0.png)

3. Карточки гостиниц расположены не по сетке:
![image](https://user-images.githubusercontent.com/8547830/153491911-8eee6b63-9604-4a27-8a7e-2d84c9bbc529.png)

4. Панинация и фильтрация так же не по сетке:
![image](https://user-images.githubusercontent.com/8547830/153492035-7eb207cb-f694-4cc0-8b56-cd26b82d53fc.png)

5. Ошибки валидации нужно исправить:
![image](https://user-images.githubusercontent.com/8547830/153492374-2ffbab64-f79d-4ec2-97b9-818efc8fb65c.png)

ParovatkinA committed:
исправляет ошибку в футере

для правильного отображения информации футера по центру

ParovatkinA committed:
форматирует код для валидации

ParovatkinA committed:
добавляет флексовую модель для класса advantages-item и изменяет расположение двух картинок в "преимуществах"

ParovatkinA committed:
удаляет ненужные пробелы

ParovatkinA committed:
Создает новый div для двух преимуществ (1 и 5)

обертываем <h3> и <p>

ParovatkinA committed:
удаляет ненужные стили

ParovatkinA committed:
добавляет новый класс для первого и второго преимущества

чуть ранее переопределили направление главной оси, чтобы картинка во втором преимуществе была слева

ParovatkinA committed:
cоздает флекс контейнеры на странице каталога

mpodaniev commented:
Ошибки валидации не исправленны

ParovatkinA committed:
проверяет валидность кода на странице каталога

удаляет два закрывающих тега div (c.251 и с.266)
